package.json

```diff
@@ -22,7 +22,7 @@
     "build-testnet": "npm run copy-testnet-config & npm run build",
     "build-mainnet": "npm run copy-mainnet-config & npm run build",
     "build-placeholder": "npm run copy-placeholder-config & npm run build",
-    "postbuild": "node src/scripts/createVersionFile/index",
+    "postinstall": "node src/scripts/createVersionFile/index",
```

Comment:
but if there is no install, there will be no new version file
build is always run